### PR TITLE
Fix/utility namespace

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc @fjvallarino 
Dependant PR for fix/utility-namespace

**Comment by alex-deaconu** [comment]
@RV-Adrian is the CS resolution necessary? Otherwise, looks good 👍 . Thanks.

File: bower.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "angular": "1.2.18",
     "angular-ui-router": "^0.2.15",
-    "common-style": "https://github.com/Rise-Vision/common-style.git",
+    "common-style": "https://github.com/Rise-Vision/common-style.git#v1.4.0",
```
Comment (alex-deaconu) [inline bower.json]: @RV-Adrian this is the resolution I was talking about in the other PRs. Having this here will propagate issues to other repos and force them to add `common-style` resolutions. My suggestion is however to update this bower component (and relevant code) to reference `rv-common-style`. Thanks.

Comment (RV-Adrian) [inline bower.json]: @alex-deaconu Ah, I see what you mean now. Will do!

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good 👍 . Thanks!

**Pull request merged**